SetupScript.sh

```diff
@@ -125,6 +125,8 @@ mv /opt/Epsilon-Raspberry/QT_CFLAGS_DBUS.patch /home/pi/qt5
 	-prefix /home/pi/qt5 |& tee "output_configure.txt")
 (cd /home/pi/qt5/qtbase && make |& tee "output_make.txt")
 (cd /home/pi/qt5/qtbase && make install |& tee "output_make_install.txt")
+mv /usr/bin/qmake /usr/bin/qmake.bak
+rm /usr/bin/qmake
```

Comment:
Don't need this now, since qmake got renamed to qmake.bak, this rm command won't do anything.